### DISCUSSION TITLE
Keyboard height changes when switching layers

### DISCUSSION THREAD

**Discussion by ursi** [florisboard/florisboard#1692] [Feedback]
On 0.3.14, when I switch to the symbols layer, the keyboard shrinks because I have the number row turned on. This is very jarring in messaging apps where the placement of the text is dependent on keyboard height. On 0.3.13, the rows on the symbols layer would be given more space between them as to not change the height of the keyboard. It would be nice to be able to turn this behaviour back on.

**Comment by patrickgold** [comment]
This is correct and not intended. See #1658, it is already fixed and will be released in 0.3.15-beta01 soon.
